### PR TITLE
Home/steps

### PR DESCRIPTION
**Pull request by AnnaSidiakina**
Add  section Steps (Етапи  нашої роботи) to Home page.

### CODE REVIEW

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import { useState, useEffect } from 'react';
+import { Swiper, useSwiper } from 'swiper/react';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+import 'swiper/css/scrollbar';
+import { useRef } from 'react';
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:7]: Better to split imports
1.libs
free space
2. cunstom components

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import Image from 'next/image';
+import step1 from '@public/images/homePage/steps/step1.svg';
+import step2 from '@public/images/homePage/steps/step2.svg';
+import step3 from '@public/images/homePage/steps/step3.svg';
+import step4 from '@public/images/homePage/steps/step4.svg';
+import step5 from '@public/images/homePage/steps/step5.svg';
+import step6 from '@public/images/homePage/steps/step6.svg';
+import step7 from '@public/images/homePage/steps/step7.svg';
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:15]: make index file to export there all photos and then import it here from index file 
Знімок екрана 2023-05-11 о 22 35 33
Знімок екрана 2023-05-11 о 22 34 54

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    { image: step3, description: 'Затверджуємо ТЗ', id: 3 },
+    { image: step4, description: 'Розпочинаємо розробку', id: 4 },
+    { image: step5, description: 'Тестуємо функціональність сайту', id: 5 },
+    { image: step6, description: 'Публікуємо сайт в мережі', id: 6 },
+    {
+      image: step7,
+      description: 'Отримуємо ще одного задоволеного клієнта',
+      id: 7,
+    },
+  ];
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:64]: use separate file for constant values and than import it here

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+  // };
+
+  // const SlidePrevButton = () => {
+  //   const handleClick = () => {
+  //     if (swiperRef.current) {
+  //       swiperRef.current.slidePrev();
+  //     }
+  //   };
+  //   return <button onClick={handleClick}>prev</button>;
+  // };
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:86]: Dont leave dead code

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+  useEffect(() => {
+    if (mousePosition.x > screen.width - (screen.width / 100) * 14) {
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:89]: What does mean 100 and 14 and why they are not constants

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+      swiperRef.current.slideNext();
+    } else {
+      swiperRef.current.slidePrev();
+    }
+  }, [mousePosition]);
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:94]: Use reverse audit condition and dont use else block
read airbnb convention doc in useful links

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
+  return (
+    <>
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:97]: why u use here empty tags?

File: src/components/Portfolio/ProjectInfo/Slider/Slider.jsx [hunk 8]
```diff
@@ -12,7 +12,7 @@ import {
   StyledSwiper,
   StyledSwiperSlide,
   SwiperContainer,
-} from "./Slider.styled";
+} from './Slider.styled';
```
Comment (painlalala) [inline src/components/Portfolio/ProjectInfo/Slider/Slider.jsx:15]: separate imports

File: src/pages/index.js [hunk 9]
```diff
@@ -43,6 +44,7 @@ export default function Home() {
         {/* <p>Home page</p> */}
         <HomePart1 />
         <HomePart2 />
+        <Part6Steps />
```
Comment (painlalala) [inline src/pages/index.js:47]: User semantic names

File: src/styles/Theme.jsx [hunk 10]
```diff
@@ -72,7 +73,10 @@ export const theme = {
     xxl: '32px',
     xxxl: '36px',
     xxxxl: '48px',
-    xxxxxl: '64px',
+    xxxxl: '50px',
+    xxxxxl: '56px',
+    xxxxxxl: '64px',
+    xxxxxxxl: '96px',
```
Comment (forze961) [inline src/styles/Theme.jsx:79]: Need a better name for params

File: src/styles/GlobalStyles.jsx [hunk 11]
```diff
@@ -1,6 +1,7 @@
-import { createGlobalStyle } from "styled-components";
+import { createGlobalStyle } from 'styled-components';
 
 const GlobalStyles = createGlobalStyle`
+
```
Comment (forze961) [inline src/styles/GlobalStyles.jsx:4]: Need to align whitespaces

File: src/components/common/Container/Container.styled.js [hunk 12]
```diff
@@ -1,5 +1,5 @@
-import styled from "styled-components";
-import { theme } from "@component/styles/Theme";
+import styled from 'styled-components';
+import { theme } from '@component/styles/Theme';
```
Comment (forze961) [inline src/components/common/Container/Container.styled.js:2]: Duplicate import

File: src/components/Portfolio/ProjectInfo/Slider/Slider.jsx [hunk 13]
```diff
@@ -43,9 +43,9 @@ export const Slider = ({ images }) => {
   return (
     <SwiperContainer position="relative">
       <StyledSwiper
-        onSwiper={(swiper) => (swiperRef.current = swiper)}
-        onSlideChange={() => console.log("slide change", swiperRef.current)}
-        slidesPerView={"auto"}
+        onSwiper={swiper => (swiperRef.current = swiper)}
+        onSlideChange={() => console.log('slide change', swiperRef.current)}
```
Comment (forze961) [inline src/components/Portfolio/ProjectInfo/Slider/Slider.jsx:47]: Not critical, but we shouldn't use console.log in prod

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.styled.js [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+export const StyledSpan = styled.span`
+  background-image: url(/images/header/blue.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+  color: ${({ theme }) => theme.backgroundColor.white};
+  padding: 0 18px;
+`;
+
+export const StyledSlider = styled(Swiper)``;
```
Comment (forze961) [inline src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.styled.js:25]: Nested '`'

**Review by forze961 — CHANGES_REQUESTED** [review]

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx (hunk 1 above)
Comment (AnnaSidiakina) [reply src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:7]: done

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx (hunk 2 above)
Comment (AnnaSidiakina) [reply src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:15]: done

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx (hunk 3 above)
Comment (AnnaSidiakina) [reply src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:64]: done

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx (hunk 5 above)
Comment (AnnaSidiakina) [reply src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:89]: done

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx (hunk 6 above)
Comment (AnnaSidiakina) [reply src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:94]: I am not sure that I got it right, but I removed else block

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx (hunk 7 above)
Comment (AnnaSidiakina) [reply src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.jsx:97]: not anymore

File: src/components/Portfolio/ProjectInfo/Slider/Slider.jsx (hunk 8 above)
Comment (AnnaSidiakina) [reply src/components/Portfolio/ProjectInfo/Slider/Slider.jsx:15]: This is Vladimir's file, I will tell him about it

File: src/pages/index.js (hunk 9 above)
Comment (AnnaSidiakina) [reply src/pages/index.js:47]: done

File: src/styles/Theme.jsx (hunk 10 above)
Comment (AnnaSidiakina) [reply src/styles/Theme.jsx:79]: Well, I have an idea how to make it better. But the thing is that we have now 15 fontSizes. I will talk with designers, maybe we can cut some of them

File: src/styles/GlobalStyles.jsx (hunk 11 above)
Comment (AnnaSidiakina) [reply src/styles/GlobalStyles.jsx:4]: done

File: src/components/common/Container/Container.styled.js (hunk 12 above)
Comment (AnnaSidiakina) [reply src/components/common/Container/Container.styled.js:2]: I can't see it

File: src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.styled.js (hunk 14 above)
Comment (AnnaSidiakina) [reply src/components/Home/HomeParts/HomePart6Steps/HomePart6Steps.styled.js:25]: removed

**Review by forze961 — APPROVED** [review]

**Pull request merged**